### PR TITLE
QE 4.13 Zonal GA

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/cucushift-installer-rehearse-vsphere-ipi-zones-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-zones
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/cucushift-installer-rehearse-vsphere-ipi-zones-chain.yaml:2]: I think we do not have to introduce this chain, but copy the following plain ref step to provision chain.

Comment (sgaoshang) [reply ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/cucushift-installer-rehearse-vsphere-ipi-zones-chain.yaml]: Fixed

**Pull request merged**